zeroconf/test.py

```diff
@@ -657,14 +657,59 @@ def test_bad_service_names(self):
         for name in bad_names_to_try:
             self.assertRaises(r.BadTypeInNameException, self.browser.get_service_info, name, 'x.' + name)
 
+    def test_good_names_for_get_service_info(self):
```

Comment:
I think this test can be dropped. Isn't it fully redundant with `test_good_instance_names_without_protocol`?

Reply:
Will remove as long as coveralls gives the 👍